src/main/java/calculator/OperatorType.java

```diff
@@ -0,0 +1,40 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.function.BiFunction;
+
+public enum OperatorType {
+	ADDITION("+", Integer::sum),
+	SUBTRACTION("-", (val1, val2) -> val1 - val2),
+	MULTIPLY("*", (val1, val2) -> val1 * val2),
+	DIVIDE("/", (val1, val2) -> {
+		if (val2 == 0) {
+			throw new ArithmeticException("0으로 나눌 수 없습니다.");
+		}
+		return val1 / val2;
+	});
+
+	private final String symbol;
+	private final BiFunction<Integer, Integer, Integer> expression;
+
+	OperatorType(final String symbol, final BiFunction<Integer, Integer, Integer> expression) {
```

Comment:
```suggestion
	OperatorType(final String symbol, final BinaryOperator<Integer> expression) {
```


오토 박싱을 방지하기위해 위와 같이 사용할 수도 있으니 참고바라요,
 `@FunctionalInterfaces should be as specialised as possible`